docs/glossary.md

```diff
@@ -263,9 +257,9 @@ Multi-Cluster Replication is a feature which asynchronously replicates Workflow
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
-#### [Multi-Cluster Replication](/clusters#multi-cluster-replication)
+#### [Multi-region Namespace](/namespaces#multi-region-namespace)
 
-Multi-Cluster Replication is a feature which asynchronously replicates Workflow Executions from active Clusters to other passive Clusters, for backup and state reconstruction.
+A Multi-region is a Namespace that exists across Clusters when Multi-Cluster Replication is set up.
```

Comment:
Or have multiple regions selected when you create the Namespace in Cloud.

<img width="630" alt="Screenshot 2024-04-23 at 3 57 23 PM" src="https://github.com/temporalio/documentation/assets/34380806/a4094f6c-c61c-4155-8a14-85df3a52a8a2">

We will want to call this out in the section above too I think.

Reply:
There are two definitions in-play. One for Multi-Cluster Replication, another for Multi-region Namespaces. We can combine them, etc. But I think this may fall outside the scope of this ticket without a specific ask from Cloud.

Thoughts?